### ISSUE TITLE
Transaction name not quoted

### ISSUE THREAD

**Issue by mokkabonna** [ratanakvlun/sequelize-msnodesqlv8#10]
In: https://github.com/ratanakvlun/sequelize-msnodesqlv8/commit/f02a35ccc6e413f14c8d84ca5f093b934ae06951#diff-259dec4414a400a6e895f16ff1d0ca3b you removed the quoting of name, this fails when the name is dashed.

This is the query that is executed for me using sequelize ^4.10.0:

```sql
SAVE TRANSACTION [10c95e3640360232c791-savepoint-1];
ROLLBACK TRANSACTION [10c95e3640360232c791-savepoint-1];
ROLLBACK TRANSACTION;
```
And then I get the following error:

> SequelizeDatabaseError: [Microsoft][SQL Server Native Client 11.0][SQL Server]Incorrect syntax near 'c95e3640360232c791'

As you can see also the first rollback uses the hypenated name. Something like this should work:

```js
name = '"' + name + '"'
```

**Comment by ratanakvlun** [comment]
Double quotes only work if QUOTED_IDENTIFIER is ON for the database. You are right though; the issued command should delimit the name and I will change it to add brackets (which should work as delimiter whether QUOTED_IDENTIFIER is ON or OFF). Are you sure those commands are what is being executed? Because brackets should allow hyphens to be used.

**Comment by ratanakvlun** [comment]
Ah, I think I see what's going on. Sequelize is assuming the database module is putting in brackets and so it outputs a log message with the brackets when in reality the database module was not using any delimiter.

**Comment by mokkabonna** [comment]
Ah I see, yeah that was the output printed at least.

**Comment by ratanakvlun** [comment]
Should be resolved in 0.2.6-beta.7

**Comment by mokkabonna** [comment]
Great! Thanks for the quick response.
